### ISSUE TITLE
Show current path in app title

### ISSUE THREAD

**Issue by Suor** [Guake/guake#665]
This is useful for time logging apps, they can see what you were doing. This used to work in some older versions.

**Comment by Stibbons** [comment]
which app title? In the tab? there is an option for this

**Comment by Suor** [comment]
Now app titke is just `Guake!`. I want it to include path there, this is not the same as tab names.

**Comment by Stibbons** [comment]
Indeed, this is normal. It has been like this since the begining; I set this as a feature request to change the app title

**Comment by Stibbons** [comment]
If you want this feature, please vote on [FeatHub](https://feathub.com/Guake/guake)
